src/App/Resources/zh-TW/Resources.resw

```diff
@@ -242,6 +242,9 @@
   <data name="ConnectButton.Content" xml:space="preserve">
     <value>連線到自訂 IP 和連接埠</value>
   </data>
+  <data name="ConnectedTo" xml:space="preserve">

```

Comment:
Don't edit the zh-tw & zh-cn resource files. They are autogenerated by the touchdown build.

https://www.osgwiki.com/wiki/Factory_Orchestrator_Developer_Information#Localization_Build